tasks/utils.py

```diff
@@ -280,3 +280,14 @@ def load_release_versions(ctx, target_version):
             # environment when running a subprocess.
             return {str(k):str(v) for k, v in versions[target_version].items()}
     raise Exception("Could not find '{}' version in release.json".format(target_version))
+
+def go111module_envvar(command):
+    # tests if the GO111MODULE environment variable is set to on; if yes, stops
+    # the build because the Datadog Agent can't be built with go modules.
```

Comment:
although we're not enforcing this in our invoke task codebase, let's prefer python-style inline documentation here, i.e.:

```suggestion
    """
    Test if the GO111MODULE environment variable is set to on; if so, stop
    the build because the Datadog Agent can't be built with go modules.
    """
```